src/Wallabag/UserBundle/Entity/User.php

```diff
@@ -240,4 +240,14 @@ public function isTrustedComputer($token)
 
         return false;
     }
+
+    public function serialize()
+    {
+        return serialize($this->id);
```

Comment:
I don't remember exactly but I guess it was required for something. Please test without it.